Google.Api.Generator.Tests/ProtoTests/ResourceNames/ResourceNames.proto

```diff
@@ -11,75 +11,101 @@ import "google/api/field_behavior.proto";
 service ResourceNames {
   option (google.api.default_host) = "resourcenames.example.com";
 
-  // Test top-level resource definition.
+  // Test simple resource definition.
   rpc SimpleMethod(SimpleResource) returns(Response) {
     option (google.api.method_signature) = "name";
   }
 
-  // Test resource definition inlined in the proto message.
-  rpc SimpleInlineMethod(SimpleInlineResource) returns(Response) {
+  // Test simple resource reference.
```

Comment:
Possibly single rather than simple, to be consistent in terminology everywhere?

Reply:
Yes, I meant "Simple" meaning that this was the simplest case. But "Single" is now better.
Done.